robot/Cumulus/resources/BDI_API.py

```diff
@@ -0,0 +1,22 @@
+from tasks.salesforce_robot_library_base import SalesforceRobotLibraryBase
+from tasks.set_BDI_mapping_mode import SetBDIMappingMode
+
+
+class BDI_API(SalesforceRobotLibraryBase):
+    def configure_BDI(self, mode):
+        return self._run_subtask(SetBDIMappingMode, mode=mode)
+
+    def _get_di_mode(self):
+        token = self.get_org_namespace_prefix()
```

Comment:
I think you want self.get_npsp_namespace_prefix(). In an un-namespaced org with managed NPSP installed, the NPSP objects still have a namespace prefix.

Reply:
https://github.com/SalesforceFoundation/Cumulus/pull/4592/files#diff-5a22133e151c83a83d77f9f305c4bfdbR10